src/effects/ColorAverage.tsx

```diff
@@ -2,7 +2,7 @@ import { ColorAverageEffect, BlendFunction } from 'postprocessing'
 import React, { Ref, ForwardRefExoticComponent, forwardRef, useMemo } from 'react'
 
 export type ColorAverageProps = Partial<{
-  blendFunction: number
+  blendFunction?: BlendFunction
```

Comment:
`Partial` type already makes properties optional. No need for `?` :D

ts doc: https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype